src/jit/emitxarch.cpp

```diff
@@ -2375,7 +2378,7 @@ void                emitter::emitIns(instruction ins)
             ins == INS_stosb   ||
             ins == INS_stosd   ||
             ins == INS_stosp 
-#ifdef _TARGET_AMD64_
+#ifndef LEGACY_BACKEND
```

Comment:
It (and the other AVX instructions) are not under #if FEATURE_AVX_SUPPORT in instrsxarch.h; I was just using the same #if as instrsxarch.h.